iamproperty/iamproperty.github.io
# Ds 108/action bar

## Summary of Changes:
1. CSS for drawer component
2. JS module
3. Vue component
4. Iam sold admin example page updated

## Rationale
Create a drawer component to hide away content or links but also keep them in a position of access without scrolling. This component should help the likes of the iamsold admin pages by increasing the space for the content by moving the sidebar into the drawer.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/drawer

## Ticket(s)
DS-108

## Notes
